pkg/cli/cliflags/flags.go

```diff
@@ -518,6 +518,16 @@ An IPv6 address can also be specified with the notation [...], for
 example [::1]:26257 or [fe80::f6f2:::]:26257.`,
 	}
 
+	ListenSQLMaxClients = FlagInfo{
+		Name: "sql-max-clients",
```

Comment:
`sql-max-tcp-clients` might be a clearer name? Or `sql-max-tcp-conns`? Currently it's implying total # of sql clients, which might mislead folks.

Edit: or maybe not, see my comment below.